client/scss/components/_captioned_image.scss

```diff
@@ -119,6 +125,10 @@
   &:focus {
     text-decoration: underline;
   }
+
+  &.captioned-image__truncate-control--is-hidden {
+    display: none;
```

Comment:
I think we eschewed `is-hidden` early-doors in favour of per-component, BEM-y selectors. Maybe we've changed our minds?